src/iosrv/init.c

```diff
@@ -571,6 +646,8 @@ Options:\n\
       Number of helper XS -per vos target (default 1)\n\
   --firstcore=firstcore, -f firstcore\n\
       index of first core for service thread (default 0)\n\
+  --pinned_numa_node=numanode, -p numanode\n\
+      Bind to cores within the specified NUMA node (default 0)\n\
```

Comment:
(style)  space prohibited between function name and open parenthesis '('

Reply:
Agreed to ignore.